browser-extension/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@opentelemetry/browser-extension",
+  "version": "0.1.1",
+  "description": "A browser extension that injects opentelemetry tracers into any website",
+  "scripts": {
+    "compile": "npx webpack --mode=production",
```

Comment:
I added "clean" and "prepare", not sure about version:update. Since I also extract the package name from package.json at compile time, I have the following right now in `ui/index.tsx`:

```javascript
...
const packageJson = require('../../package.json');
...
              <Typography variant="h6" color="inherit" noWrap>
                {capitalCase(packageJson.name)} ({packageJson.version})
              </Typography>
...
``` 

I could make it consistent with the other packages and work with a version.ts, I don't have a real preference.